### ISSUE TITLE
Add Neuronpedia logits

### ISSUE THREAD

**Issue by DavidUdell** [DavidUdell/sparse_circuit_discovery#144]
Now that everything's in place, I really need to see the Neuronpedia logits data in each node, not just the activation data.
